src/cli/build.rs

```diff
@@ -2,6 +2,9 @@ use std::process::Command;
 use rustc_version::version_meta;
 
 fn main() {
+    // Copy VERSION file. Do not fail e.g. when built via `cargo publish`
+    let _ = std::fs::copy("../../VERSION", "./src/VERSION");
```

Comment:
maybe we put it back old way of referring VERSION?  That way, we don't need to maintain `build.rs`

Reply:
The CLI already has a build.rs that we need to maintain. I think that this is the simplest way to handle our VERSION needs, I really (_really_) disliked having feature flags that set VERSION="UNDEFINED".

Reply:
Can you add `rerun-if-changed` this to `build.rs` ?  https://stackoverflow.com/questions/49077147/how-can-i-force-build-rs-to-run-again-without-cleaning-my-whole-project